README.md

```diff
@@ -45,11 +45,11 @@ A key-map of data related to the received payload in the form of:
 {
     type: 'pr',         // can be 'pr' or 'repo'
     hookId: '81e6bd80-9a2c-11e6-939d-beaa5d9adaf3', // webhook event uuid
-    action: 'opened',   // can be 'opened', 'closed', or 'synchronized' for type 'pr'; 'push' for type 'repo'
-    username: 'batman',
+    action: 'opened',   // can be 'opened', 'reopened', 'closed', or 'synchronized' for type 'pr'; 'push' for type 'repo'
+    username: 'batman', // should be the actor/creator of the webhook event (not necessarily the author)
```

Comment:
Let's have username as a different value than the repo owner so it's more clear.